src/goto-symex/html.cpp

```diff
@@ -0,0 +1,702 @@
+#include "util/message.h"
+#include <fstream>
+#include <goto-symex/goto_trace.h>
+#include <ostream>
+#include <sstream>
+#include <unordered_map>
+#include <list>
+#include <util/language.h>
+#include <langapi/language_util.h>
+#include <optional>
+#include <filesystem>
+#include <util/cmdline.h>
```

Comment:
Please follow the standard of the rest of esbmc here: alphabetical order, no double quotes.

Also, do you need all these includes?